app/assets/stylesheets/galleries.css

```diff
@@ -1,5 +1,6 @@
 .gallery-box { cursor: pointer; width: 20px; text-align: center; } /* The + / - symbol to minimize a gallery */
 .gallery-title { height: 5px; } /* So that when a gallery is minimized there is spacing */
+.character-icon { width: 100px; height: 100px; }
```

Comment:
When they're selected they have fixed square sizes so I was doing this for consistency.

Relevant code:

```css
.selected-icon { border: 10px #3F7055 solid; width: 80px; height: 80px; }
```

I could put it in a box and have it with a maximum size, and center it, so smaller images (including non-square images) are the appropriate size, and larger images are scaled appropriately? This would preserve the grid structure.